### PR TITLE
[OSDOCS-7396]: Remove titles from customer portal and add 'Gathering audit logs' back to ROSA/OSD

### PR DESCRIPTION
**Pull request by mletalie**
Version(s):

4.13+
Issue:

https://issues.redhat.com/browse/OSDOCS-7396
Link to docs preview:

ROSA: https://63421--docspreview.netlify.app/openshift-rosa/latest/security/audit-log-view
OSD: https://63421--docspreview.netlify.app/openshift-dedicated/latest/security/audit-log-view
QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by mletalie** [comment]
Hello @oarribas,
   As the SME for this issue, can you please review? Thanks!

**Comment by mletalie** [comment]
@yingzhanredhat, please review. Thanks! ML

**Comment by mletalie** [comment]
Received "LGTM" from QE on OSDOCS ticket.

File: security/audit-log-view.adoc [hunk 1]
```diff
@@ -1,22 +1,31 @@
...
+endif::openshift-rosa,openshift-dedicated[]
 :context: audit-log-view
 
 toc::[]
 
 {product-title} auditing provides a security-relevant chronological set of records documenting the sequence of activities that have affected the system by individual users, administrators, or other components of the system.
 
-// About the API audit log
 include::modules/nodes-nodes-audit-log-basic.adoc[leveloffset=+1]
+audit-log-view.adoc
```
Comment (jab-rh) [inline security/audit-log-view.adoc:20]: Erroneous text.

**Review by jab-rh — COMMENTED** [review]
Left a comment, thanks!

File: security/audit-log-view.adoc (hunk 1 above)
Comment (mletalie) [reply security/audit-log-view.adoc]: Corrected, thanks!

File: support/gathering-cluster-data.adoc [hunk 2]
```diff
@@ -40,8 +40,10 @@ include::modules/gathering-data-specific-features.adoc[leveloffset=+2]
 
 * link:https://access.redhat.com/support/policy/updates/openshift[Red Hat {product-title} Life Cycle Policy]
 
+ifdef::openshift-rosa,openshift-dedicated[]
```
Comment (kcarmichael08) [inline support/gathering-cluster-data.adoc:43]: I couldn't find the preview page to check out if this was properly displaying - am not familiar with the doc set and didn't find a page "gathering data about your cluster" but I might be confused. If so, please ignore.

**Review by kcarmichael08 — COMMENTED** [review]
Looks good for merge review!

**Pull request merged**